packages/block-library/src/heading/heading-toolbar.js

```diff
@@ -9,6 +9,7 @@ import { range } from 'lodash';
 import { __, sprintf } from '@wordpress/i18n';
 import { Component } from '@wordpress/element';
 import { Toolbar } from '@wordpress/components';
+import { AlignmentToolbar } from '@wordpress/block-editor';
 
 class HeadingToolbar extends Component {
```

Comment:
I think that component was just meant to render the heading size contro. Can we just render the alignment toolbar in the `edit.js` file directly instead?

Reply:
Yes, I agree that it should be handled in `edit.js`. `HeadingToolbar` should be called `HeadingSizeToolbar` and handle only that part of the interactions.

Reply:
I moved the align toolbar to the proper location :)